src/modules/PaintBucket/PaintBucket.js

```diff
@@ -24,7 +24,7 @@ module.exports = exports = function(pixels, options) {
     minFactor = (1 - tolerance / 100);
   fillColor = fillColor.substring(fillColor.indexOf('(') + 1, fillColor.length - 1); // extract only the values from rgba(_,_,_,_)
   fillColor = fillColor.split(',');
-    
+  fillColor[3] =  fillColor[3] * 256;
```

Comment:
Shouldn't this be multiplied by `255` and not `256`?

Reply:
Also, why is it multiplied by some factor in the first place?

Reply:
> Shouldn't this be multiplied by `255` and not `256`?

because range is 0-255

Reply:
> Also, why is it multiplied by some factor in the first place?

Because input of alpha channel that we were getting was between 0-1

I searched but wasn't able to found the exact formula

Reply:
Oh yeah! The alpha is between 0 and 1.

Reply:
If that is the case then it needs to be multiplied by 255 not 256.

Reply:
Yea you're right :sweat_smile: my mistake !
@Olusamimaths please make the requested changes